cluster/images/kubemark/Dockerfile

```diff
@@ -12,8 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# The line below points to distroless/base as of 2021-06-29. The SHA should be
-# kept in sycn with distroless_base definition in the WORKSPACE file.
```

Comment:
I have no idea what "the WORKSPACE file" is. I hope the revised comment is okay.

"Update periodically" is vague, but I don't have any better recommendation.